<html>
<head>
  <title>Rumen Kostadinov</title>
</head>
<body>
  <pre>
Rumen Kostadinov, Ph.D.
Research Associate
Pediatric Oncology
Laboratory of Patrick A. Brown
Sidney Kimmel Comprehensive Cancer Center
Johns Hopkins University
</pre>
</br>
<pre>
Currently, I develop computational methods for analysis of 
whole genome DNA, RNA, and DNA methylation sequencing data 
from infant MLL-rearranged leukemias.

Current software (under development):
-------------------------------------
RNA sequencing data analysis:
- rsem/STAR/EBseq/limma-voom analysis pipeline

DNA bisulfite sequencing data analysis:
- epitoolkit: a suite of C++ programs for analysis of WGBS data (with some tools also working on WGS and RNA-seq data)
- Expectation maximization (EM) algorithm for methylation-defined subclonal deconvolution


Previously, I developed software for phylogenetic analysis and
copy number analysis to characterize clonal evolutionary dynamics in Barrett’s esophagus.

Past software (completed):
--------------------------------
Phylogenetic analysis tools:  
- BEAST modification: additions of 0/1 binary evolutionary model, sequence error model, and a cenancestor node
  to allow inference from copy number and LOH biopsy data
- PAUP analysis: modeling copy number and LOH changes for maximum parsimony inference

Simulation:
- Cryptsim: Spatial stochastic tumor evolution simulator that keeps track of phylogenies
- SerialSimCoal modification: addition of a LOH/copy number evolutionary model to a coalescent simulator

Illumina SNP array analysis tools:
- DNA copy number data from SNP arrays analysis pipeline that
  includes impute2 for allele-specific changes, and segmentation with thresholds for calling
  copy number and LOH events.  
</pre>
    
Publications:
<ol>  
<li> Kostadinov R, Maley CC, Kuhner MK. Bulk Genotyping of Biopsies Can Create Spurious Evidence for Hetereogeneity in Mutation Content. PLoS Comput Biol. Public Library of Science; 2016;12: e1004413. doi:10.1371/journal.pcbi.1004413</li>
<li> Somarelli JA, Ware KE, Kostadinov R, Robinson JM, Amri H, Abu-Asab M, et al. PhyloOncology: Understanding cancer through phylogenetic analysis. Biochim Biophys Acta - Rev Cancer. 2016; doi:10.1016/j.bbcan.2016.10.006</li>
<li> Kostadinov RL, Kuhner MK, Li X, Sanchez CA, Galipeau PC, Paulson TG, et al. NSAIDs Modulate Clonal Evolution in Barrett’s Esophagus. Absher D, editor. PLoS Genet. Public Library of Science; 2013;9: e1003553. doi:10.1371/journal.pgen.1003553</li>
<li> Li X, Galipeau PC, Paulson TG, Sanchez CA, Arnaudo J, Liu K, et al. Temporal and spatial evolution of somatic chromosomal alterations: A case-cohort study of Barrett’s esophagus. Cancer Prev Res (Phila). 2013; 1940–6207.CAPR–13–0289–. doi:10.1158/1940-6207.CAPR-13-0289</li>
<li> Martens E, Kostadinov R, Maley C, Hallatschek O. Spatial structure increases the waiting time for cancer. New J Phys. 2011;13: 115014. doi:10.1088/1367-2630/13/11/115014</li>
<li> Reid BJ, Kostadinov R, Maley CC. New Strategies in Barrett’s Esophagus: Integrating clonal evolutionary theory with clinical management. Clin Cancer Res. 2011;17: 3512–3519. doi:10.1158/1078-0432.CCR-09-2358</li>
<li> Lai LA, Kostadinov R, Barrett MT, Peiffer DA, Pokholok D, Odze R, et al. Deletion at Fragile Sites Is a Common and Early Event in Barrett’s Esophagus. Mol cancer Res. 2010;8: 1084–1094. doi:10.1158/1541-7786.MCR-09-0529</li>
<li> Kostadinov R, Malhotra N, Viotti M, Shine R, D’Antonio L, Bagga P. GRSDB: a database of quadruplex forming G-rich sequences in alternatively processed mammalian pre-mRNA sequences. Nucleic Acids Res. 2006;34: D119–24. doi:10.1093/nar/gkj073</li>
</ol>
</body>
</html>
<!-- Thanks for checking out the source code -->
