<html>
<head>
  <title>Rumen Kostadinov</title>
</head>
<body>
<pre>
Rumen Kostadinov, Ph.D.
Research Associate
Pediatric Oncology
Laboratory of Patrick A. Brown
Sidney Kimmel Comprehensive Cancer Center
Johns Hopkins University

GitHub source code repositories <a href=https://github.com/rkostadi>https://github.com/rkostadi</a>

Currently, I develop computational methods for analysis of 
whole genome DNA, RNA, and DNA methylation sequencing data 
from infant MLL-rearranged leukemias.

Current software (under development):
-------------------------------------
RNA sequencing data analysis:
- rsem/STAR/EBseq/limma-voom analysis pipeline

DNA bisulfite sequencing data analysis:
- epitoolkit: a suite of C++ programs for analysis of WGBS data (with some tools also working on WGS and RNA-seq data)
- Expectation maximization (EM) algorithm for methylation-defined subclonal deconvolution


Previously, I developed software for phylogenetic analysis and
copy number analysis to characterize clonal evolutionary dynamics in Barrettâ€™s esophagus.

Past software (completed):
--------------------------
Phylogenetic analysis tools:  
- BEAST modification: additions of 0/1 binary evolutionary model, sequence error model, and a cenancestor node
  to allow inference from copy number and LOH biopsy data
- PAUP analysis: modeling copy number and LOH changes for maximum parsimony inference

Simulation:
- Cryptsim: Spatial stochastic tumor evolution simulator that keeps track of phylogenies
- SerialSimCoal modification: addition of a LOH/copy number evolutionary model to a coalescent simulator

Illumina SNP array analysis tools:
- DNA copy number data from SNP arrays analysis pipeline that
  includes impute2 for allele-specific changes, and segmentation with thresholds for calling
  copy number and LOH events.  
</pre>
</body>
</html>
<!-- Thanks for checking out the source code -->
